web/impact/impact/v1/views/base_list_view.py

```diff
@@ -39,30 +37,29 @@ def get(self, request):
         }
         return Response(result)
 
-    def _results(self, limit, offset):
+    def results(self, limit, offset):
         queryset = self.helper_class.all_objects()
         updated_at_after = self.request.query_params.get(
             'updated_at.after', None)
         updated_at_before = self.request.query_params.get(
             'updated_at.before', None)
         if updated_at_after or updated_at_before:
-            queryset = _filter_by_date(queryset,
-                                       updated_at_after,
-                                       updated_at_before)
+            queryset = self._filter_by_date(queryset,
+                                            updated_at_after,
+                                            updated_at_before)
         count = queryset.count()
         return (count,
-                [self.helper_class(obj).serialize()
+                [self.serialize(obj)
                  for obj in queryset[offset:offset + limit]])
 
-
-def _filter_by_date(queryset, updated_at_after, updated_at_before):
-    updated_at_after = parse_date(updated_at_after)
-    updated_at_before = parse_date(updated_at_before)
-    if updated_at_after:
-        queryset = queryset.filter(Q(updated_at__gte=updated_at_after))
-    if updated_at_before:
-        queryset = queryset.exclude(Q(updated_at__gt=updated_at_before))
-    return queryset
+    def _filter_by_date(self, qs, after, before):
+        updated_at_after = parse_date(after)
+        updated_at_before = parse_date(before)
+        if updated_at_after:
+            qs = qs.filter(Q(updated_at__gte=updated_at_after))
```

Comment:
Why are these filters using `Q` objects? Should work fine as 
`qs = qs.filter(updated_at__gte=updated_at_after)`

Reply:
Fixed.